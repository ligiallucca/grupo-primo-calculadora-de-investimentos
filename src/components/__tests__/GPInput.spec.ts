import { describe, it, expect } from 'vitest'

import { mount } from '@vue/test-utils'
import GPInputVue from '../GPInput.vue'

describe('GPInput', () => {
    it('renders properly', () => {
        const wrapper = mount(GPInputVue, {
            modelValue: {
                type: String,
                default: "",
            },
        })
        expect(wrapper.html).toMatchSnapshot();
    })
})