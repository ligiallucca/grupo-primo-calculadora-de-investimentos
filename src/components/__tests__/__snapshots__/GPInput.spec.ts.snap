// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`GPInput > renders properly 1`] = `[Function]`;
